setup.py

```diff
@@ -39,7 +39,7 @@
 ]
 extras = {
     "bqstorage": [
-        "google-cloud-bigquery-storage >= 0.6.0, <2.0.0dev",
+        "google-cloud-bigquery-storage >= 0.8.0, <2.0.0dev",
```

Comment:
Independent of this change, but we should consider bumping this to a 1.x now that we have a GA v1.

Reply:
I don't see this published on [PyPI](https://pypi.org/project/google-cloud-bigquery-storage/#history) yet? I used `0.8.0` because it was the most recent version at that time.

Reply:
Sorry, I wasn't clear.  I meant we should do a release of storage to bump the major version to indicate the stability level more clearly now that v1 is GA.  I'll see about getting this done tomorrow.